lib/fuzzer/main.zig

```diff
@@ -0,0 +1,508 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const mutate = @import("mutate.zig");
+const InputPool = @import("input_pool.zig").InputPool;
+const MemoryMappedList = @import("memory_mapped_list.zig").MemoryMappedList;
+const feature_capture = @import("feature_capture.zig");
+
+const Allocator = std.mem.Allocator;
+const ArrayList = std.ArrayList;
+const ArrayListUnmanaged = std.ArrayListUnmanaged;
+const Options = std.testing.FuzzInputOptions;
+const Prng = std.Random.DefaultPrng;
```

Comment:
Just use `std.Random`, methods will change if DefaultPrng changes and things will break